tests/test_utils.py

```diff
@@ -6,13 +6,15 @@
 
 from contextlib import contextmanager
 
+import ddt
```

Comment:
i think you need to add `ddt` to the requirements? this is causing all your tests to fail :)